.github/workflows/release.yml

```diff
@@ -2,109 +2,48 @@
 name: release
 
 on:
-  pull_request:
   push:
+    tags:
+      - "v20*"
 
 jobs:
-  build-distribution:
-    name: Build catalystcoop.pudl distribution for PyPI
+  create-github-release:
+    name: Create GitHub Release
     runs-on: ubuntu-latest
-    if: github.event.pull_request.head.repo.fork == false
+    permissions:
+      contents: write
     steps:
       - name: Checkout source
         uses: actions/checkout@v6
         with:
-          fetch-depth: 2
-      - name: Set up Python
-        uses: actions/setup-python@v6
-        with:
-          python-version: "3.13"
-      - name: Build source and wheel distributions
+          fetch-depth: 0
+      - name: Create GitHub Release
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
         run: |
-          python -m pip install --upgrade build twine
-          python -m build
-          twine check --strict dist/*
-      - name: Store the distribution packages
-        uses: actions/upload-artifact@v6
-        with:
-          name: python-package-distributions
-          path: dist/
+          gh release create '${{ github.ref_name }}' \
+            --title 'PUDL ${{ github.ref_name }}' \
+            --notes "PUDL is an application that must be installed from the repository using pixi.
```

Comment:
This is a placeholder. Ideally we would extract the relevant release notes from `release_notes.rst` and use them here automatically, plus some basic pointers to install & data access.